app/views/compute_resources/form/_vmware.html.erb

```diff
@@ -4,9 +4,14 @@
 <% datacenters = list_datacenters f.object%>
 <%= selectable_f(f, :datacenter, datacenters, {}, {:label => _('Datacenter'), :help_inline_permanent => load_datacenters_button_f(f, !datacenters.empty?) })%>
 <%= text_f f, :pubkey_hash, :disabled => true, :label => _("Fingerprint"), :size => "col-md-8" %>
+<%= select_f f, :console_type, f.object.supported_console_types,
+  :first, :last, {}, {
+  :label => _('Console Type'),
+  :help_inline => _('VNC consoles are unsupported on VMware ESXi 6.5 and later.')
```

Comment:
Yeah, we can easily find that out. But it needs a connection to vSphere to work.

```
ComputeResource.find_by_name('Example').send(:client).vsphere_rev
=> "6.0"
```

But I think, this is something for http://projects.theforeman.org/issues/17727